src/Sulu/Bundle/HttpCacheBundle/Resources/varnish/sulu.vcl

```diff
@@ -83,6 +83,12 @@ sub sulu_backend_response {
 }
 
 sub sulu_deliver {
+    if (!resp.http.X-Cache-Debug) {
```

Comment:
Not sure if this is the correct place?

Reply:
I would say that is the correct place. If you rebase that PR and everything still works as expected, we could probably merge that pretty soon.